Missing "&" in mappings.go file line 69

patluri43 opened the pull request.

patluri43 committed:
commiting changes for pre_existing Auth server resources

patluri43 committed:
commiting changes for pre_existing Auth server resources

patluri43 committed:
commiting changes for pre_existing Auth server resources

jralmaraz committed:
Skeleton/DRAFT for Profile mappings

jralmaraz committed:
Skeleton/DRAFT for Profile mappings

quantumew committed:
Merge pull request #304 from jralmaraz/master

Skeleton/DRAFT for Profile mappings

quantumew committed:
Pairing with @jralmaraz on #246

quantumew committed:
Pairing with @jralmaraz on #246

quantumew committed:
Committing progress

quantumew committed:
Committing progress

patluri43 committed:
updated SDK with missing "&" in mappings.go file line 69

patluri43 committed:
updated SDK with missing "&" in mappings.go file line 69

patluri43 committed:
Merge branch 'master' of https://github.com/patluri43/terraform-provider-okta

patluri43 committed:
deleted local changes

patluri43 committed:
readding deleted files

patluri43 committed:
merging the changes from articulate repo

Merge branch 'master' of https://github.com/patluri43/terraform-provider-okta

patluri43 committed:
commiting the files from articulate repo

patluri43 committed:
commiting the files

patluri43 committed:
commiting changes realted to claims and policies

quantumew commented:
Hey @patluri43, looks like there are some unrelated changes on this PR so I am going to leave this. I will however cherry pick that pointer fix. While that mapping stuff is still not tested or merged to master, that would have definitely been an issue so great catch! Finally getting some time now to wrap it up if all goes well.

Commit:
Document need of lifecycle argument when using `app_group_assignment` and `app_user`

When managing application group assignments via `app_group_assignment` resource
the application resource's group management must be disabled using `lifecycle.ignore_changes`
attribute to avoid removal of externally added assignments. Same applies
to `app_user` resource.

While this behaviour is shown in examples, it is best to warn users by
adding this to docs too.

See #322

quantumew committed:
Merge pull request #327 from zoltanpeto/docs-add-lifecycle-note

Document the need for lifecycle argument

vijetm committed:
Adds support for basic auth application

vijetm committed:
Addresses Andy's comments

quantumew committed:
Merge pull request #329 from vijetm/features/basic-auth

Adds support for basic auth application

quantumew committed:
Merge pull request #324 from articulate/feature/mappings

okta_profile_mapping resource

quantumew committed:
v3.0.33

quantumew committed:
Use pointers for zeroable values fixes #330

Test the zero value of max password attempts

Install latest version

Vendor modules

quantumew committed:
Merge pull request #331 from articulate/bugfix/policy-zero-values

Use pointers for values that can be zero

quantumew committed:
v3.0.34

vijetm committed:
Adds support to create users with credentials

vijetm committed:
Adds user password update functionality

vijetm committed:
Support updating recovery credentials

alkar committed:
Changes buildSchema to not modify its arguments

Fixes #338: when invoked through `buildRuleSchema`, it modifies
`userExcludedSchema` resulting in the schema for the different rule
types be mixed.

Signed-off-by: Dimitrios Karagiannis <dhkarag@gmail.com>

vijetm committed:
Adds a new test case for update credentials

quantumew committed:
Require target_id on profile mapping

Also add data source to allow users to determine target/source ids

Update CHANGELOG

Update new data source to be more specific

All other profile mapping sources are do not have unique names

Update find profile mapping source to take type as well

Update docs

quantumew committed:
Merge pull request #341 from articulate/feature/support-target-id

Require target_id on profile mapping

quantumew committed:
Merge pull request #339 from alkar/buildschema-do-not-modify

Changes buildSchema to not modify its arguments

quantumew committed:
v3.0.36

quantumew committed:
Safely check for 404 when retrieving app

quantumew committed:
Merge pull request #344 from articulate/bugfix/safely-check-for-404

Safely check for 404 when retrieving app

quantumew committed:
Update changelog

quantumew committed:
Ensure we sync schema index so it shows up in plan on import

quantumew committed:
Sync it to the value in the config

quantumew committed:
Rather then syncing index, just set it on import directly

quantumew committed:
v3.0.37

quantumew committed:
Merge pull request #346 from articulate/bugfix/sync-index-for-imported-schemas

Ensure we sync schema index so it shows up in plan on import

quantumew committed:
Merge pull request #334 from vijetm/enhancement/user-with-credentials

Adds support to create users with credentials

quantumew committed:
v3.0.38

jnathanh committed:
Switch id with type in import documentation

The documentation incorrectly shows that the okta id is used to import email templates. The current implementation uses the email template type.

I tested with ids and it failed, then with types and they succeeded.

quantumew committed:
Merge pull request #354 from jnathanh/jnathanh-patch-1

Switch id with type in import documentation

patluri43 committed:
Merge remote-tracking branch 'upstream/master'

patluri43 committed:
commiting the deleted files

patluri43 committed:
commiting the mappings

patluri43 committed:
commiting the mapping files

ocxo commented:
Thanks for the contribution @patluri43! This repo has officially moved to Hashicorp's collection of Terraform Providers at https://github.com/terraform-providers/terraform-provider-okta/. If you still wish to see this PR land on the provider, please re-open at https://github.com/terraform-providers/terraform-provider-okta/pulls.
